test/testing/fakeauth_test.js

```diff
@@ -27,7 +28,7 @@ testApi.declare({
 
 
 suite('fakeauth', function() {
-  var fakeauth = require('../../src/fakeauth');
+  var fakeauth = require('../../lib/fakeauth');
```

Comment:
Hmm, good question are these tests compiled then lib otherwise src

I suspect they are compiled...
Note: there is some subtle bugs in Babel particularly array. Prototype. Includes doesn't get handled correctly.... With Babel compile.. So testing with lib is safest..